ui/app/controllers/jobs/index.js

```diff
@@ -88,7 +84,7 @@ export default Controller.extend(Sortable, Searchable, {
     const allNames = this.visibleJobs.mapBy('name');
     const nameHistogram = allNames.reduce((hist, name) => {
       if (hasPrefix.test(name)) {
-        const prefix = name.match(/(.+?)[-.]/)[1];
+        const prefix = name.match(/(.+?)[-._]/)[1];
```

Comment:
Hey, sorry late to the game here, just noticed a tinsy tiny nitpick, but potentially important for someone skimming the code.

Line 79

```javascript
// A prefix is defined as the start of a job name up to the first - or .
// ex: mktg-analytics -> mktg, ds.supermodel.classifier -> ds
```

^ that specifically states `- or .`, if you are going to keep the comment here I'd probably update that to include the `_`

Actually just looking further while I'm here. Hoisting the at least second regex out of the `reduce` hotpath might also be an option here. Not sure how hot this can get, if at all, but I think this is compiling a new regex for every loop? Usually I try and hoist all RegEx patterns into the top level scope as they can be a 'performance thing'. It might be nice to have them literally one after the other in the code also, that would make any future tiny differences like this even more unlikely? (Not sure if you can make a pattern you can reuse here, but you could also dynamically make a RegExp() that used the same `[.-_]`, if you like that. Theoretically slower than the literal, but if you are only doing it once maybe it doesn't matter)

Anyway, all just suggestions!